### PR TITLE
Add drawBitmap method

### PR DESCRIPTION
**Pull request by hrules6872**

### CODE REVIEW

**Comment by alorma** [comment]
Wow! That looks so cool!

Let me check how it works, and i will merge it

**Comment by hrules6872** [comment]
I made some changes which they could break something... Maybe you should increase the major version ;)

**Comment by hrules6872** [comment]
screenshot

**Comment by alorma** [comment]
Did you tested with Glide, Picasso, or similar?

**Comment by alorma** [comment]
I tested with glide and there are some problems.

It should be a ImageView instead of View, so we can use it everywhere, overriding setImageDrawable / seImageBitmap etc...

Could you do that?

**Comment by hrules6872** [comment]
I don't see any advantage of using an ImageView... what problems did you
see?

File: timeline/src/main/java/com/alorma/timeline/RoundTimelineView.java [hunk 1]
```diff
@@ -26,41 +30,45 @@ public RoundTimelineView(Context context, AttributeSet attrs, int defStyleAttr,
...
     @Override
-    protected void drawInternalEnd(Canvas canvas, Paint paintInternal, float centerX, float centerY,
+    protected void drawInternal(Canvas canvas, Paint paintInternal, float centerX, float centerY,
         float size) {
         drawCircle(canvas, centerX, centerY, size, paintInternal);
     }
 
+    @Override protected void drawBitmap(Canvas canvas, float left, float top, int size) {
+        if (internalBitmapCache == null) {
+            internalBitmapCache = transform(((BitmapDrawable) internalDrawable).getBitmap(), size);
```
Comment (alorma) [inline timeline/src/main/java/com/alorma/timeline/RoundTimelineView.java]: Please, don't cast directly, check if it's BitmapDrwable, and if not, get the bitmap

**Comment by alorma** [comment]
Basically, with a `ImageView`you can use it everywhere, with other libraries as i  mentioned, glide, picasso... And it will work "out of the box"

**Comment by hrules6872** [comment]
Done :)

**Comment by alorma** [comment]
Greaaaaaat!!! :dancer:

**Comment by alorma** [comment]
Great job man!

**Pull request merged**